eth/downloader/downloader.go

```diff
@@ -1663,6 +1681,15 @@ func (d *Downloader) processFastSyncContent(latest *types.Header) error {
 }
 
 func splitAroundPivot(pivot uint64, results []*fetchResult) (p *fetchResult, before, after []*fetchResult) {
+	if len(results) == 0 {
+		return
```

Comment:
Pls never use naked returns, it' very hard to follow what's being returned (especially when they are already modified)